src/database/migrations/zz-create-triggers.js

```diff
@@ -54,50 +53,48 @@ EXECUTE PROCEDURE update_beneficiaries_community_states();`);
         SELECT SUM(claimed + NEW.amount) INTO beneficiary_claimed FROM beneficiary WHERE "communityId"=community_public_id AND address=NEW.address;
         UPDATE beneficiary SET claimed = beneficiary_claimed WHERE "communityId"=community_public_id AND address=NEW.address;
         -- update total claimed
-        SELECT SUM(claimed + NEW.amount) INTO state_claimed FROM ubi_community_state WHERE "communityId"=NEW."communityId";
-        UPDATE ubi_community_state SET claimed = state_claimed WHERE "communityId"=NEW."communityId";
+        -- SELECT SUM(claimed + NEW.amount) INTO state_claimed FROM ubi_community_state WHERE "communityId"=NEW."communityId";
+        -- UPDATE ubi_community_state SET claimed = state_claimed WHERE "communityId"=NEW."communityId";
```

Comment:
just remove everything we don't use anymore

Reply:
done